tests/data/split_merge/step4_split_groups_array/catalogs/mycatalog/catalog/groups/00000__group/controls/00001__control.json

```diff
@@ -217,7 +217,7 @@
                                 "value": "c."
                             }
                         ],
-                        "prose": "Require {{ ac-2_prm_1 }} for group and role membership;"
+                        "prose": "Require {{ insert: param, ac-2_prm_1 }} for group and role membership;"
```

Comment:
This is crazy - created an issue for it here: https://github.com/usnistgov/OSCAL/issues/971

No action on our part except for with the SSP code. Can you open an issue on it please?

Reply:
done